tests/components/anthropic/test_config_flow.py

```diff
@@ -108,7 +108,7 @@ async def test_options(
                 ),
                 body={"type": "error", "error": {"type": "invalid_request_error"}},
             ),
-            "invalid_request_error",
+            "unknown",
```

Comment:
`invalid_request_error` is not currently available in `strings.json`, and I don't think it would add anything over a regular `unknown` error.